packages/table/src/Cell/Cell.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { css, cx } from '@leafygreen-ui/emotion';
+
+import {
+  alignmentStyles,
+  baseCellStyles,
+  baseTableSidePadding,
+  cellContentContainerStyles,
+  cellContentTransitionStyles,
+} from './Cell.styles';
+import { CellProps } from '.';
+
+const Cell = ({ className, align, children, ...rest }: CellProps) => (
```

Comment:
The standalone Cell is rendered when the user isn't using RT, so this is basically doing what `InternalCellBase` used to do in the old implementation. And since this component renders a `td` all the time, the browser will surface that the component shouldn't be used outside a `tr` - an error that  says a `td` cannot be a child of a `div` for example.